### PR TITLE
Update script to use V2 pipeline

### PR DESCRIPTION
**Pull request by jaelim-ms**
Improve merge build pipeline. 45 mins -> 25 mins.

### CODE REVIEW

**Comment by jaelim-ms** [comment]
The merge requires build pipeline configuration change to use V2.

**Comment by jaelim-ms** [comment]
Valid point @guperrot but we need to merge this one and switch the definition to new one to verify it.

File: vsts/publish-podspec.sh [hunk 1]
```diff
@@ -66,6 +66,17 @@ if [ "$mode" == "internal" ] || [ "$mode" == "test" ]; then
 
   if [ "$mode" == "test" ]; then
 
+    # Read publish version for current build
+    version=$(cat $CURRENT_BUILD_VERSION_FILENAME)
+
+    # Exit if response doesn't contain an array
```
Comment (clpolet) [inline vsts/publish-podspec.sh]: Does it really contain an array, not just the version?

Comment (jaelim-ms) [reply vsts/publish-podspec.sh]: Copy-paste. 😄

**Pull request merged**